onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -863,6 +879,28 @@ def _where_input_wrangler(
         matcher=lambda sample: "weight" not in sample.kwargs,
         reason="this Aten overload need weight as kwargs",
     ),
+    skip(
+        "nn.functional.scaled_dot_product_attention",
+        matcher=lambda sample: (attn_mask := sample.kwargs.get("attn_mask")) is not None
+        and attn_mask.dtype == torch.bool,
+        reason="this overload takes a non-boolean mask",
+    ),
+    skip(
+        "nn.functional.scaled_dot_product_attention",
+        matcher=lambda sample: sample.kwargs.get("dropout_p") != 0.0,
```

Comment:
dropout will behave the same way eval or training. Ops are all functional so if an op has a training behavior it will have an argument controlling that behavior, I believe.

Reply:
Let me check dropout

Reply:
Yeah: for example aten::dropout itself has `dropout(Tensor input, float p, bool train) -> Tensor`

Reply:
It appears `scaled_dot_product_attention` is used internally and has a more functional interface. The `dropout_p` is set by caller (e.g., `multi_head_attention_forward`, multi_head_attention_forward) depending on training mode.

Reply:
Nice!